scene/resources/curve.cpp

```diff
@@ -1388,29 +1395,35 @@ real_t Curve3D::interpolate_baked_tilt(real_t p_offset) const {
 		return baked_tilt_cache.get(0);
 	}
 
-	int bpc = baked_tilt_cache.size();
 	const real_t *r = baked_tilt_cache.ptr();
 
 	if (p_offset < 0) {
 		return r[0];
 	}
 	if (p_offset >= baked_max_ofs) {
-		return r[bpc - 1];
+		return r[pc - 1];
 	}
 
-	int idx = Math::floor((double)p_offset / (double)bake_interval);
-	real_t frac = Math::fmod(p_offset, bake_interval);
-
-	if (idx >= bpc - 1) {
-		return r[bpc - 1];
-	} else if (idx == bpc - 2) {
-		if (frac > 0) {
-			frac /= Math::fmod(baked_max_ofs, bake_interval);
+	int start = 0, end = pc, idx = (end + start) / 2;
```

Comment:
We tend to prefer keeping each variable declaration on its own line, especially when there's calculations involved like here:
```suggestion
	int start = 0;
	int end = pc;
	int idx = (end + start) / 2;
```
(same below)